mozci/console/commands/push.py

```diff
@@ -551,6 +551,23 @@ def handle(self) -> None:
                     )
                     self.errors[push] = e
 
+            all_pushes = set(
+                [push] + [child for child in push._iterate_children(max_depth=50)]
+            )
+            all_failures_classifications = [
+                set([c for c, _ in group.classifications])
+                for p in all_pushes
+                for group in p.group_summaries.values()
+            ]
+            if push.backedout and all(
+                failure_classifications == {"intermittent"}
+                for failure_classifications in all_failures_classifications
+            ):
+                self.line(
+                    f"<comment>Push {push.branch}/{push.rev} was backedout and all of its failures and the ones of its children were marked as intermittent by Sheriffs</comment>"
```

Comment:
```suggestion
                    f"<comment>Push {push.branch}/{push.rev} was backedout and all of its failures and the ones of its children were marked as intermittent</comment>"
```

We want to warn when classifications from mozci are all intermittent

Reply:
Are you talking about autoclassifications or all `MozciClassification` stored in Treeherder?

Reply:
> > where all failures on the push itself and its children are marked as intermittent;
>
> Eva - Should I use the max_depth attribute to list children (so we don't end up fetching 500+ pushes if we are evaluating a really old push)?
> Eva - What does marked as intermittent precisely mean? Classified as intermittent by mozci or marked by a Sheriff?

> Joel - my best guess at answering your question is you should use max_depth, Ideally we would start at a given revision and work from there- I would be 50 revisions would be safe enough for 99% of the situations
> Joel - and I am not sure about marked as intermittent I assume it would mean by sheriffs, but as time goes on it would probably need to include both sheriffs and high confidence mozci

We talked about this with Joel on Matrix while you were on PTO and we didn't quite figure it out apparently

Reply:
Thinking about it more, we could also just check all classifications from Treeherder, no matter where they are coming from (sheriffs or the mozci hook).

Reply:
We just need to take into account both "intermittent" and "autoclassified intermittent" (but if we do the solution with _iterate_failures I proposed below, we can ignore this aspect anyway)